RoboFile.php

```diff
@@ -902,4 +911,15 @@ private function execWithXDebug($command) {
     $php_config->usePersistent();
     return $exitCode;
   }
+
+  private function createDoctrineEntityManager() {
+    define('ABSPATH', getenv('WP_ROOT') . '/');
+    \MailPoet\Config\Env::$db_prefix = '';
```

Comment:
To be precise, I should set it to _something_ if it was not set already. It's only because of [this](https://github.com/mailpoet/mailpoet/pull/2164/files/c31f861213da2d8ffe4aba36b75b2b75917a5a37..e647b376b6b4d2fa72517e217afd72564abe8a87#diff-e757190306843f9a937a0e2180f2733eR19), the prefix is not compiled anywhere. I can put there some kind of IF to avoid the side effects.

Reply:
Fixed.